src/api/mock/authenticationAPI.js

```diff
@@ -0,0 +1,35 @@
+import { DELAY } from "../config";
+
+const USERS = [
```

Comment:
since the register is working I guess you don't need this data.

Reply:
This data is for the mock API. We were thinking that having the mock APIs will make testing fast. We will not need to wait for the backend to respond, by simply setting the delay to 0.

Reply:
Which mock API I though all users should be registered on the staging API since the app is set up and already running?

Reply:
When demonstrating to someone, we shall use the real API, but when running the tests, we use the mock, especially on Travis, we use the mocks.

Reply:
So its just used for testing?